src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/AssessmentEntity.kt

```diff
@@ -19,6 +19,23 @@ class AssessmentEntity (
         val createdDate: LocalDateTime? = null,
 
         @Column(name = "COMPLETED_DATE")
-        val completedDate: LocalDateTime? = null
+        val completedDate: LocalDateTime? = null,
```

Comment:
Are assessments not co-terminus with the period of supervision?

Reply:
It should be but it seems like something we would want to know so it's more of a convenience field because it gives us the current assessment for an offender without needing to make an external call to find a supervision ID and also allows us to do ordering by completed date and will help with displaying history and graphs I think. I envisage it will be set in response to an event coming out of one of the other services to say the offender is no longer under supervision.

Reply:
Fair enough :)